src/test/kotlin/no/nav/syfo/kuhrsar/api/KuhrsarApiSpek.kt

```diff
@@ -0,0 +1,183 @@
+package no.nav.syfo.kuhrsar.api
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.ktor.http.*
+import io.ktor.http.HttpHeaders.Authorization
+import io.ktor.server.testing.*
+import io.mockk.*
+import no.nav.emottak.subscription.*
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.ereg.api.*
+import no.nav.syfo.testhelper.*
+import no.nav.syfo.util.bearerHeader
+import org.amshove.kluent.shouldBeEqualTo
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class KuhrsarApiSpek : Spek({
+    val objectMapper: ObjectMapper = apiConsumerObjectMapper()
+
+    describe(KuhrsarApiSpek::class.java.simpleName) {
+
+        with(TestApplicationEngine()) {
+            start()
+
+            val externalMockEnvironment = ExternalMockEnvironment()
+
+            beforeGroup {
+                externalMockEnvironment.startExternalMocks()
+            }
+
+            afterGroup {
+                externalMockEnvironment.stopExternalMocks()
+            }
+
+            application.testApiModule(
+                externalMockEnvironment = externalMockEnvironment,
+            )
+
+            val urlKuhrsar = kuhrsarProxyBasePath
+            val subscriptionMock = externalMockEnvironment.subscriptionMock
+            clearSubscriptionMock(subscriptionMock)
+
+            describe("Find samhandler") {
+                val validToken = generateJWT(
+                    externalMockEnvironment.environment.aadAppClient,
+                    externalMockEnvironment.wellKnown.issuer,
+                    testPadm2ClientId,
+                )
+                val kuhrsarRequest = KuhrsarRequest(
+                    behandlerIdent = PersonIdent("01010112345"),
+                    partnerId = 1,
+                    legekontorOrgName = "navn",
+                    legekontorHerId = "herid",
+                    data = "data".toByteArray(),
+                )
+                describe("Happy path") {
+
+                    it("should return OK if request is successful, but samhandler not found") {
```

Comment:
superpirk: fjerne ordet"should" på disse testene? Should er utydelig, det *kan* være en instruksjon, men det kan også være en spådom, eller bare et råd.